sniffles.wdl

```diff
@@ -0,0 +1,73 @@
+version 1.0
+
+workflow sniffles {
+    input {
+        File sortedBamFile
+        String outputFileNamePrefix = basename(sortedBamFile, ".bam")
+        String? additionalParameters
+    }
+    parameter_meta {
+        sortedBamFile: "Input bam file (must be sorted)"
+        outputFileNamePrefix: "Variable used to set the outputfile name"
+        additionalParameters: "Additional parameters to be added to the sniffles command"
+    }
+
+    meta {
+        author: "Matthew Wong"
+        email: "m2wong@oicr.on.ca"
```

Comment:
this email address will disappear once you leave OICR.  absolutely fine to include your name, but i don't think it makes sense to use this address.